package.json

```diff
@@ -59,13 +59,13 @@
     "@angular/platform-browser": "~4.1.3",
     "@angular/platform-browser-dynamic": "~4.1.3",
     "@angular/router": "^5.1.3",
-    "@types/fullcalendar": "^3.5.0",
     "@types/jasmine": "2.5.36",
     "@types/jquery": "^3.2.15",
     "@types/node": "^6.0.46",
     "camelcase": "^4.0.0",
     "concurrently": "3.4.0",
     "core-js": "^2.4.1",
+    "fullcalendar": "^3.8.0",
```

Comment:
may be @types/fullcalendar ?

Reply:
But if updating fullcalendar to latest version. It is throwing (ERROR in node_modules/ng-fullcalendar/src/calendar.component.d.ts(3,10): error TS2305: Module ''fullcalendar'' has no exported member 'Options'.)

Reply:
@jimmylogic, in fullcalendar@3.8.0 don't export Option parametr. I will be do it in feature release 2.0.0 of ng-fullcalendar